docs/mdSyntaxHighlighter.js

```diff
@@ -0,0 +1,81 @@
+/** @typedef {import("markdown-it")} MarkdownIt */
```

Comment:
This isn't the right location for this file. Can you please move it to `src/_plugins` and rename to `md-syntax-highlighter.js`?

Also, did you copy any of this code from somewhere else? If so, we need to include the license information.

Reply:
Hi Nicholas, as there're some discrepancies between eslint docs site and vitepress, e.g., they use shiki for syntax highlight and author their code in TypeScript, while we use Prismjs and JavaScript, hence I have to modify the markdown plugins based on their implementation. So yes, I did copy some code from its repository.